### PR TITLE
chore: rebrand to The Sapphire Project

### PR DESCRIPTION
**Pull request by alexthemaster**

### CODE REVIEW

**Comment by alexthemaster** [comment]
This PR made changes to the LICENSE file so I kindly ask all the current contributors to react with a 👍 if they agree with the changes!

@Favna 
@kyranet 
@QuantumlyTangled

**Comment by alexthemaster** [comment]
> I assume you did so already but just checking, you did a "find in project" action for "skyra"?

Affirmative.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -13,24 +13,20 @@ in your text editor or IDE of your choice to ensure builds from GitHub Actions d
...
 
 [Discord Server]:                            https://join.skyra.pw
-[here]:                                      https://github.com/skyra-project/skyra/pulls
+[here]:                                      https://github.com/sapphire-project/api/pulls
 [ESLint]:                                    https://eslint.org/
 [Node.JS]:                                   https://nodejs.org/en/download/
 [Yarn]:                                      https://classic.yarnpkg.com/en/docs/install
 [OOP paradigms]:                             https://en.wikipedia.org/wiki/Object-oriented_programming
-[Discord Bot Best Practices]:                https://github.com/meew0/discord-bot-best-practices
-[scripts]:                                   /scripts
```
Comment (Favna) [inline .github/CONTRIBUTING.md:36]: Please revert this line

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -13,24 +13,20 @@ in your text editor or IDE of your choice to ensure builds from GitHub Actions d
...
+- Everything in API should be generally useful for the majority of users. Don't let that stop you if you've got a good concept though, as your idea still might be a great addition.
 - Everything should be shard compliant. If code you put in a pull request would break when sharding, break other things from supporting sharding, or is incompatible with sharding; then you will need to think of a way to make it work with sharding in mind before the pull request will be accepted and merged.
 - Everything should follow [OOP paradigms] and generally rely on behaviour over state where possible. This generally helps methods be predictable, keeps the codebase simple and understandable, reduces code duplication through abstraction, and leads to efficiency and therefore scalability.
 - Everything should follow our ESLint rules as closely as possible, and should pass lint tests even if you must disable a rule for a single line.
-- Everything should follow [Discord Bot Best Practices]
-- Scripts that are to be ran outside of the scope of the bot should be added to [scripts] directory and should be in the `.mjs` file format.
```
Comment (Favna) [inline .github/CONTRIBUTING.md:25]: Please revert this line. While the folder does not exist right now, it is the rule that we'll follow. Specifically the location and the fact that they should be `.mjs` files.

File: .github/SECURITY.md [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 ## Reporting a Vulnerability
 
-If you find a vulnerability in @skyra/api's codebase please report it immediately.
+If you find a vulnerability in @sapphire-project/api's codebase please report it immediately.
 If you deem the vulnerability exploitable by any users in any shape or form please join the Discord server at https://join.skyra.pw and then DM either @kyranet (kyra#0001) or @favna (Favna#0001).
```
Comment (Favna) [inline .github/SECURITY.md:12]:
```suggestion
If you deem the vulnerability exploitable by any users in any shape or form please join the Discord server at https://join.skyra.pw/sapphire and then DM either @kyranet (kyra#0001) or @favna (Favna#0001).
```

Just created the alias, might as well apply it here.

File: README.md [hunk 4]
```diff
@@ -45,9 +45,9 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+    <td align="center"><a href="https://favware.tech/"><img src="https://avatars3.githubusercontent.com/u/4019718?v=4" width="100px;" alt=""/><br /><sub><b>Jeroen Claassens</b></sub></a><br /><a href="https://github.com/sapphire-project/api/commits?author=Favna" title="Code">💻</a> <a href="https://github.com/sapphire-project/api/commits?author=Favna" title="Documentation">📖</a> <a href="#infra-Favna" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a> <a href="#projectManagement-Favna" title="Project Management">📆</a></td>
+    <td align="center"><a href="https://quantumlytangled.com"><img src="https://avatars1.githubusercontent.com/u/7919610?v=4" width="100px;" alt=""/><br /><sub><b>Nejc Drobnic</b></sub></a><br /><a href="https://github.com/sapphire-project/api/commits?author=QuantumlyTangled" title="Code">💻</a> <a href="https://github.com/sapphire-project/api/commits?author=QuantumlyTangled" title="Documentation">📖</a></td>
```
Comment (cfanoulis) [inline README.md:50]: revert this, let allcontributors do it

File: package.json [hunk 5]
```diff
@@ -1,10 +1,9 @@
 {
-	"name": "@skyra/api",
+	"name": "@spfw/api",
 	"version": "0.0.1",
-	"description": "Plugin for @skyra/framework to expose an API from an @klasa/core bot.",
+	"description": "Plugin for @spfw/framework to expose an API from an @klasa/core bot.",
 	"main": "dist/index.js",
 	"types": "dist/index.d.ts",
-	"author": "@skyra",
```
Comment (cfanoulis) [inline package.json:7]: revert this to
`"author": "@sapphire-project"`

**Comment by cfanoulis** [comment]
And as this PR introduces a license change, I need @Favna & @QuantumlyTangled to react with 👍 on this comment to show they agree with the change

File: LICENSE.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # The MIT License (MIT)
 
-Copyright © `2018-2020` `Kyra`
+Copyright © `2018-2020` `The Sapphire Project`
```
Comment (tjrgg) [inline LICENSE.md:3]: What about `The Sapphire Project and its contributors`? That'd probably be more accurate.

Comment (tjrgg) [reply LICENSE.md:3]:
```suggestion
Copyright © `2018-2020` `The Sapphire Project and its contributors`
```

**Pull request merged**